frontend/public/components/machine-config-pool.tsx

```diff
@@ -155,7 +155,7 @@ const MachineConfigPoolCounts: React.SFC<MachineConfigPoolCountsProps> = ({ obj
 
 const MachineConfigPoolSummary: React.SFC<MachineConfigPoolSummaryProps> = ({ obj }) => {
   const machineConfigSelector = _.get(obj, 'spec.machineConfigSelector');
-  const machineSelector = _.get(obj, 'spec.machineSelector');
+  const nodeSelector = _.get(obj, 'spec.nodeSelector');
   return (
     <ResourceSummary resource={obj}>
```

Comment:
Pass `showNodeSelector` to `ResourceSummary` and remove the logic for it here.

Reply:
Ok, fixed, but needed additional logic in [details-page.tsx](https://github.com/openshift/console/pull/4927/files#diff-272cf4caf4399b8a447965ce49e4543aL40).